requirements.txt

```diff
@@ -1,3 +1,19 @@
+# database
```

Comment:
I moved things in this file so it's easier to do `hashin xyz requirements.txt` and it's more likely that the item falls into the right section.